Lesson Plans/Flight Instructor/CFI/CFI1.4 Teaching Process.md

```diff
@@ -0,0 +1,250 @@
+		1. Intro: Gives a brief overview and setup a relaxed, welcoming atmosphere for discussions
+		2. Use lead-off question to open up an area
+		3. Use follow-up questions to guide the direction: dig into learner's explanation or lead the stray discussion back
+		4. Conclusion: Summarize the material covered. Tie together various points and topics discussed.
+
+#### [[Problem-Based Learning]]
+1. [[Scenario-Based Training]]
+2. Collaborative Problem-Solving
+	1. Instructor provides a problem for the group to solve it
+	2. The instructor provides assistance only when needed, but understands that learning to solve a problem without assistance is part of the process
+3. Case-Study
+		1. NTSB aviation accidents is an excellent source of real-world case studies
+
+#### e-Learning
+1. Computer-Assisted Learning (CAL)
+	1. Use computer / tablet with software
+	2. FAA test prep guide software is useful for preparing for knowledge tests
+2. Simulation, Video Games
+	1. Simulators (BATD, AATD, Full-motion Simulator)
+	2. Video Games (X-plane, MSFS, etc.)
+
+#### Cooperative or Group Learning
+1. Organize learners into small groups to maximize understanding
+2. Instructor needs to plan early to determine what group should be able to do on their own
+3. Conditions and controls:
+	1. Small, heterogeneous groups
+	2. Clean, complete instructions: What to do, in what order, with what materials
+	3. Individual accountability
+	4. Recognition and reward for group success
+	5. Time after completion for learners to systematically reflect upon how they worked together as a team
+
+#### Demonstration-Performance
+1. Best used with mental or physical skills that requires practice
+2. Five Phases: Explanation, Demonstration, Learner Performance, Instructor Supervision and Evaluation
+
+#### Drill and Practice
+1. Law of exercise: Connections are strengthened with practice
+
+#### Application and Assessment
+1. The learner should have use of the skills they learned: Forming good habits and filter out faulty habits
+2. Each lesson should conclude with an assessment
+	1. Review of what has been covered
+	2. Ask learners to demonstrate how well the objectives has been met with
+	3. It can be used to inform the next lesson and/or logged formally and tracking learner's progress
+
+#### Instructional Aid
+1. Devices that assist an instructor in the teaching-learning process
+2. Key Theory
+	1. Covers the key points & concepts
+	2. Straightforward and factual
+	3. Best if relatively simple
+3. Reasons for the use
+	1. Help gain and hold learners' attention
+	2. Help with understanding and retention
+	3. Solves certain language barrier problems
+	4. Clarify relationships between material objects and concepts
+	5. Saves time
+4. [[Guideline for Use of Instructional Aids]]
+5. Types of Instructional Aids
+	1. Marker Board
+	2. Supplemental Printed Material
+	3. Enhanced Training Materials
+	4. Projected Material
+	5. Video
+	6. Interactive Systems
+	7. Computer-Assisted Learning
```

Comment:
EVERYTHING IS COMPUTER!

Reply:
see my point of AIH being repetitive too? lol